src/status_im/ui2/screens/chat/home.cljs

```diff
@@ -129,7 +129,9 @@
                      (re-frame/dispatch [:accept-all-activity-center-notifications-from-chat chat-id]))
     :on-long-press #(re-frame/dispatch [:bottom-sheet/show-sheet
                                         {:content (fn []
-                                                    [sheets/actions home-item])}])}])
+                                                    [actions/actions
+                                                     (:chat-type home-item)
+                                                     (:chat-id home-item)])}])}])
```

Comment:
Is it better to destructure `chat-type` and I think chat-id is already destructured in the `render-fn` definition?